source/blog/2017-09-27-webadmin-lldp.html.md

```diff
@@ -0,0 +1,30 @@
+---
+title: LLDP information in webadmin
+author: amusil
+tags: webadmin, networking
+date: 2017-09-27 09:00:00 CET
+---
+
+In the oVirt 4.2.0 we have introduced support for LLDP protocol. 
+LLDP stands for Link Layer Discovery Protocol, which is protocol used by network devices for advertising identity and capabilities to neighbors on a LAN. [More information about LLDP](https://learningnetwork.cisco.com/docs/DOC-26851).
+
+READMORE
+
+### Useful information
+
+The information gathered by protocol can be used for better network configuration. UI shows most important informations e.g. System name, Port description and VLAN IDs. Administrator can set VLAN ID according to switch configuration. In case of some problem Port description can become handy as it shows switch port number to which is network interface connected.
+
+
+### Where to find this information
+
+In the **Administration Portal**. From the vertical menu select **Compute** > **Hosts** and click on **Link (Name)** of your desired host. Then select **Network Interfaces** > **Setup Host Networks**. By hovering over interfaces appropriate tooltil will show with it's information. 
```

Comment:
show with it's information -> show with **its** information 
Why not: 
... show with its information like this: ![screen](../images/blog/2017-09-27/1.png)